specification/mysql/resource-manager/Microsoft.DBforMySQL/FlexibleServers/preview/2024-12-01-preview/openapi.json

```diff
@@ -0,0 +1,7043 @@
+      "properties": {
+        "state": {
+          "$ref": "#/definitions/AdvancedThreatProtectionState",
+          "description": "Specifies the state of the Advanced Threat Protection, whether it is enabled or disabled or a state has not been applied yet on the specific database or server."
+        }
+      },
+      "required": [
+        "state"
+      ]
+    },
+    "Azure.ResourceManager.ArmResponse<BackupAndExportResponse>": {
+      "type": "object",
+      "description": "Azure operation completed successfully.",
+      "properties": {
+        "body": {
+          "$ref": "#/definitions/BackupAndExportResponse",
+          "description": "The body type of the operation request or response."
+        }
+      },
+      "required": [
+        "body"
+      ]
+    },
+    "AzureADAdministrator": {
+      "type": "object",
+      "description": "Represents a Administrator.",
+      "properties": {
+        "properties": {
+          "$ref": "#/definitions/AdministratorProperties",
+          "description": "The properties of an administrator.",
+          "x-ms-client-flatten": true
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ]
+    },
+    "Backup": {
+      "type": "object",
+      "description": "Storage Profile properties of a server",
+      "properties": {
+        "backupRetentionDays": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Backup retention days for the server."
+        },
+        "backupIntervalHours": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Backup interval hours for the server."
+        },
+        "geoRedundantBackup": {
+          "type": "string",
+          "description": "Whether or not geo redundant backup is enabled.",
+          "default": "Disabled",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "x-ms-enum": {
+            "name": "EnableStatusEnum",
+            "modelAsString": true,
+            "values": [
+              {
+                "name": "Enabled",
+                "value": "Enabled"
+              },
+              {
+                "name": "Disabled",
+                "value": "Disabled"
+              }
+            ]
+          }
+        },
+        "earliestRestoreDate": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Earliest restore point creation time (ISO8601 format)",
+          "readOnly": true
+        }
+      }
+    },
+    "BackupAndExportRequest": {
+      "type": "object",
+      "description": "BackupAndExport API Request",
+      "properties": {
+        "targetDetails": {
+          "$ref": "#/definitions/BackupStoreDetails",
+          "description": "Backup Target Store Details"
+        }
+      },
+      "required": [
+        "targetDetails"
```

Comment:
in old swagger:
```
    "BackupAndExportRequest": {
      "type": "object",
      "properties": {
        "targetDetails": {
          "$ref": "#/definitions/BackupStoreDetails"
        }
      },
      "required": [
        "backupSettings",
        "targetDetails"
      ],
      "allOf": [
        {
          "$ref": "#/definitions/BackupRequestBase"
        }
      ]
    },
```
I don't think `backupSettings` should be set here, this is from `BackupStoreDetails`